workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-node/.nvmrc

```diff
@@ -0,0 +1 @@
+v16.18.1
```

Comment:
`.nvmrc` seems out of place here, especially because liferay-portal's node version if 16.13.0. If you need to really make sure it is a certain version, you can specify that in the package.json "engines" field instead

Reply:
Don't need it, lets remove it.